openedx/core/djangoapps/user_api/urls.py

```diff
@@ -25,9 +26,21 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^v1/me$', ME, name='own_username_api'),
-    url(r'^v1/accounts/{}$'.format(settings.USERNAME_PATTERN), ACCOUNT_DETAIL, name='accounts_api'),
-    url(r'^v1/accounts$', ACCOUNT_LIST, name='accounts_detail_api'),
+    url(
```

Comment:
I agree with the `legacy_urls` file, but here I was editing it to be aligned with the style right below it, a style I decided to continue with the addition of `RegistrationValidationView`'s endpoint.